lib/newsletter_exclusion.rb

```diff
@@ -0,0 +1,93 @@
+module NewsletterExclusions
```

Comment:
Since is not including anything on any class I would rename this to a class rather than a module.